examples/arithmetic/tests/bindings/test_arithmetic.kts

```diff
@@ -13,6 +13,15 @@ try {
 assert(sub(4u, 2u) == 2uL)
 assert(sub(8u, 4u) == 4uL)
 
+assert(div(8u, 4u) == 2uL)
+
+try {
+    div(8u, 0u)
+    throw RuntimeException("Should have panicked when dividing by zero")
+} catch (e: Exception) {
+    // It's okay!
```

Comment:
IIUC this will catch the `RuntimeException` thrown above, meaning the test will always pass even if the rust code didn't panic at all. Should we catch a more specific type of error here?